### DISCUSSION TITLE
Chia eats all disk space on C:

### DISCUSSION THREAD

**Discussion by JavaMaxim** [Chia-Network/chia-blockchain#8642] [Support]
Hi,

Now I have about 5,5GB of `blockchain_wallet_v1_mainnet_nnn.sqlite` and 21GB `blockchain_v1_mainnet.sqlite` ds file sizes. Free space on disk C: is up to 20GB.

Then, when I start Chia application, after couple of minutes it eats all free disk space (zero). 

1. Is it a normal thing?
2. How much space in total does it need currently on disk C:?
3. Is it possible to places these db files on other disk?

Thanks for your help.

**Comment by emlowe** [comment]
1) yes
2) The blockchain is under constant growth so the DB is also growing
3) yes. You can edit config.yaml with a different location. You can move the entire .chia directory with CHIA_ROOT, and some people use symlink

**Comment by BrandtH22** [comment]
@JavaMaxim , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
